### DISCUSSION TITLE
Would pp or score be prioritized when retroactively rewarding pp for rate adjusted mods?

### DISCUSSION THREAD

**Discussion by developomp** [ppy/osu#27882] [Q&A] [answered]
Consider the following two scores set on the same map.

|map|pp|score|
|-|-|-|
|https://osu.ppy.sh/scores/2676820123|319pp|909,888|
|https://osu.ppy.sh/scores/2552504838|282pp|958,394|

One score has higher pp but lower score than the other. In this case, how much pp would I get rewarded when the devs eventually decide to retroactively reward pp for scores with rate adjust mods?

**Comment by peppy** [comment, marked as answer]
No idea. I doubt those values you specify are even accurate.

**Reply by respektive** [reply]
fwiw, the pp values specified by op are accurate.
However, since the linked scores are different mod combinations, they should both be preserved anyway.
